ios/Podfile

```diff
@@ -34,7 +34,6 @@ target 'PrivateKit' do
   pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/glog.podspec'
   pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'
 
-  pod 'react-native-webview', :path => '../node_modules/react-native-webview'
```

Comment:
why remove this?

Reply:
See my comment #1. Must be automatic mistake